components/vf-social-links/vf-social-links.njk

```diff
@@ -0,0 +1,50 @@
+<svg display="none" class="vf-icon-collection vf-icon-collection--social">
```

Comment:
Do we need `title` attributes on these for screen readers?

Reply:
I'll have a look - the SVG is only used for the other SVGs in the links. 

And writing that they probably need aria-role="hidden"